apps/safe-claiming-app/src/components/steps/Distribution/index.tsx

```diff
@@ -51,6 +51,7 @@ const Distribution = ({ handleNext, handleBack }: Props) => {
             <PercentageWrapper
               sx={{
                 backgroundColor: ({ palette }) => palette.safeGreen.main,
+                color: "#121312",
```

Comment:
For static values, you could also use the color palette directly i.e.

```
import palette from "./colors"
...
color: palette.text.primary
```

Reply:
we can call just
```css
color: 'text.primary'
```
without the import, no?

Reply:
No because those Percentage colors are the same in light and dark mode, but text.primary is not the same in both modes.